statannotations/Annotator.py

```diff
@@ -78,7 +78,7 @@ class Annotator:
 
     def __init__(self, ax, pairs, plot='boxplot', data=None, x=None,
                  y=None, hue=None, order=None, hue_order=None,
-                 engine="seaborn", verbose=True, **plot_params):
+                 engine="seaborn", verbose=True, show_non_significant=True, **plot_params):
```

Comment:
I would rather have a `hide_non_significant=False`, the default being to show, and the action we want to enable is hide, if that makes sense.

Reply:
I don't have a strong opinion about this and it is an easy refactor.